sycl/include/CL/sycl/ONEAPI/reduction.hpp

```diff
@@ -72,6 +74,9 @@ using IsReduBitAND = detail::bool_constant<
 
 template <typename T, class BinaryOperation>
 using IsReduOptForFastAtomicFetch =
+#ifdef SYCL_REDUCTION_NO_FAST_OPTS
```

Comment:
How does this change relate to multiple reduction support?

Reply:
Ops, sorry, that was used for debugging to compare the current 1-var implementation with the new generic case used with 1 var.
I removed that locally, just forgot to force-push once again right before MR creation. Will remove it shortly.